v1/website/versioned_docs/version-1.0.15/guides-custom-pages.md

```diff
@@ -34,12 +34,108 @@ root-of-repo
 │   └── static
 ```
 
+
+## Codeblocks in multiple languages
+
+DOCUSAURUS_CODE_TABS
+TAB_TITLE=Javascript
+```js
+const a = 33 + 4;
+console.log(a);
+```
+END_TAB
+TAB_TITLE=Python
+```py
+a = 33 + 4
+print(a)
+```
+END_TAB
+END_DOCUSAURUS_CODE_TABS
+
 Of course, you are also free to write your own pages. It is strongly suggested that you at least have an index page, but none of the pages provided are mandatory to include in your site. More information on how to use the provided components or include your own can be found [here](api-pages.md). Information on how to link to your different pages in the header navigation bar can be found [here](guides-navigation.md).
 
 > If you want your page to show up in your navigation header, you will need to update `siteConfig.js` to add to the `headerLinks` element. e.g., `{ page: 'about-slash', label: 'About/' }`,
 
 ## Adding Static Pages
 
+DOCUSAURUS_CODE_TABS
```

Comment:
I am not sure you need this all this example here if you add the documentation to https://github.com/facebook/Docusaurus/blob/master/docs/api-doc-markdown.md